cpp/src/arrow/memory_pool.h

```diff
@@ -35,44 +36,62 @@ namespace internal {
 ///////////////////////////////////////////////////////////////////////
 // Helper tracking memory statistics
 
-class MemoryPoolStats {
- public:
-  MemoryPoolStats() : bytes_allocated_(0), max_memory_(0) {}
-
-  int64_t max_memory() const { return max_memory_.load(); }
-
-  int64_t bytes_allocated() const { return bytes_allocated_.load(); }
-
-  int64_t total_bytes_allocated() const { return total_allocated_bytes_.load(); }
+class alignas(/*CacheLineSize=*/64) MemoryPoolStats {
```

Comment:
This is the alignas of stats, not the alignas of member?

Reply:
The members will start at a 64-byte boundary. The members have alignment that's lower than 64 — int64 are 8-byte aligned.

Reply:
Sure. Pushing soon.